### DISCUSSION TITLE
SB350 buffering issues with both VLC and MXPlayer (running the MP on a Fire TV Stick)

### DISCUSSION THREAD

**Discussion by DVoorhees1957** [GerryDazoo/Slinger#261] [General]
I'm still trying to find an app that will work well running on my Fire TV Stick. VLC has noticeable audio lag, pretty much right from the start of the stream. Both VLC and MXPlayer have buffering issues - the video stream stops for a second or two (or five) every 10-20 seconds.

I've tried fiddling with setting in the config.ini file, but I can't work out a good combination. Right now, I have Resolution=12, FrameRate=30, VideoBandwidth=2000, and VideoSmoothness=63. Any suggestions about what I should tweak to get a reasonably high-resolution video stream without any significant buffering?

Follow-up observations: I am able to stream, using the same config as above, to SMPlayer on my Win7 laptop with absolutely no sign of buffering. So that makes me believe that it isn't an issue with the SB350 or the network at that end of things, but rather the power of the device running the Media Player (Win7 vs. Fire TV Stick). I guess I'm looking for specific recommendations about how to get smooth streaming, without buffering, when the media player is running on a Fire TV Stick 4K...

**Comment by regmar505** [comment]
I cannot pinpoint your problem but I doubt that the Firestick is to be blamed. I have a Slingbox M2 located in Canada and a Pi3B+ server in USA and it works very well with MXPlayer on a Firestick. I even use a Resolution=15. I had the idea to connect the Slingbox to a Smart Plug so I powered it off the day before the deadline. I do not dare to put it back on, some users reported that their boxes were wiped out!
